### DISCUSSION TITLE
M1 Chip Fix / Workaround / How To Install Nightly Build 😀

### DISCUSSION THREAD

**Discussion by ChrisDev8** [AlexandreRouma/SDRPlusPlus#757] [General]
# My Story

I've had problems building the MacOS .app file for ages now. But I have a workaround.
I first went to problem #557 and used the zip file. 
There is a problem with C++ / C# Application Building I think.

# How To Install Nightly Builds:

> For those who wish to download, make sure you're logged in and go to Artifacts -> click the latest one with a check mark -> scroll 
> down to artifacts and click on sdrpp_macos_intel
> Quote from comment below

**Sorry about that just dumb**

**Comment by AlexandreRouma** [comment]
Please refrain from positing minified and/or external links to executables. 

For those who wish to download, make sure you're logged in and go to Artifacts -> click the latest one with a check mark -> scroll down to artifacts and click on `sdrpp_macos_intel`
